modules/sdk/project-templates-npm-angular-portlet/bnd.bnd

```diff
@@ -2,6 +2,7 @@ Bundle-Description: Creates a Liferay MVC portlet with npm and Angular support a
 Bundle-Name: Liferay Project Templates npm Angular Portlet
 Bundle-SymbolicName: com.liferay.project.templates.npm.angular.portlet
 Bundle-Version: 1.0.0
+-includeresource: ../project-templates/src/main/resources/definitions.vm
```

Comment:
Do we really need to embed `definitions.vm` inside `project-templates`? Since you put it in `src/main/resources`, it will become part of `project-templates`'s jar. Is it what you want?

Reply:
oh no, that isn't what I want :) don't want it in project-templates, I just was thinking of aplace to put it.  Where do you think?

Reply:
Maybe in the first template, and all the other ones will get it from it. So we don't "pollute" `project-templates` dir. Do you think it's okay?

Reply:
Yeah I would be fine with that.  We could maybe put it in the `npm-angular-portlet` one since it is the first one.

Reply:
Also we probably need to collapse more of the whitespace.  I already collapsed some whitespace in the definitions.vm but in each of the `jsp` files we have this
```
#parse("definitions.mv")

<%@ include file="/init.jsp" %
...
```

And that adds extra whitespace to the final JSP file which I don't think the user will want.  So it should probably be like

```
#parse("definitions.vm")<%@ include file="/init.jsp" %
```
so we don't add extra newlines to user's code.